src/middlewares/ValidateUser.js

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable no-tabs */
+/* eslint-disable no-unused-vars */
```

Comment:
@murediane why are we disabling eslint rules?

Reply:
it is fixed  it was from before we set up rules